lib/scholar/neighbors/random_projection_forest.ex

```diff
@@ -322,8 +313,7 @@ defmodule Scholar.Neighbors.RandomProjectionForest do
     predict_n(forest, query)
   end
 
-  defnp predict_n(forest, query) do
-    k = forest.num_neighbors
+  defn get_leaves(forest, query) do
```

Comment:
Should we add some docs here, now it is public? Or should we add `@doc false`?

Reply:
`get_leaves` is a very specific functionality that probably wouldn't be used elsewhere. So I think we might add `@doc false`